CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ### New
   - Upgraded Ruby SDK version to 2.0.0
   - Fixed add/create, delete/remove for resources
+  - Enclosure group actions
```

Comment:
"*Added support to Enclosure group actions*" seems much more like a change instead just saying "*Enclosure group actions*"